api/v1beta1/hosted_controlplane.go

```diff
@@ -178,6 +178,7 @@ const (
 	HostedControlPlaneDegraded  ConditionType = "Degraded"
 	EtcdSnapshotRestored        ConditionType = "EtcdSnapshotRestored"
 	CVOScaledDown               ConditionType = "CVOScaledDown"
+	ExternalDNSReachable        ConditionType = "ExternalDNSReachable"
```

Comment:
Can we bubble up this condition up to the HC? please declare it here https://github.com/openshift/hypershift/blob/main/api/v1beta1/hostedcluster_conditions.go#L21